ui/raidboss/data/05-shb/trial/wol-ex.js

```diff
@@ -432,21 +432,21 @@ const translate = (data, obj) => {
     {
       // Katon: San and Absolute Holy share markers
       id: 'WOLEx Spectral Ninja Cleanup',
-      netRegex: NetRegexes.startsUsing({ source: 'Spectral Ninja', id: '4EFD', capture: false }),
-      netRegexDe: NetRegexes.startsUsing({ source: 'Phantom-Ninja', id: '4EFD', capture: false }),
-      netRegexFr: NetRegexes.startsUsing({ source: 'Ninja Spectral', id: '4EFD', capture: false }),
-      netRegexJa: NetRegexes.startsUsing({ source: '幻光の忍者', id: '4EFD', capture: false }),
+      netRegex: NetRegexes.startsUsing({ source: ['Warrior Of Light', 'Spectral Ninja'], id: '4EFD', capture: false }),
```

Comment:
I had somebody say in the same party as me that they didn't get a callout.  So if you saw this, I wonder if there's some race where an existing combatant named Warrior of Light is turned into Spectral Ninja and immediately casts this and there's a race to update the name.

Can you leave a comment about why this is both?